modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -85,6 +132,53 @@ $ oc autoscale rc/<rc-name> <1>
 <2> Specify the minimum number of replicas when scaling down.
 <3> Specify the maximum number of replicas when scaling up.
 <4> Specify the target average CPU utilization over all the pods, represented as a percent of requested CPU. If not specified or negative, a default autoscaling policy is used.
++
+[TIP]
+====
+You can alternatively apply the following YAML to scale based on the percent of CPU:
+
+.Sample `HorizontalPodAutoscaler` object for scaling a replication controller on the percent of CPU utilization
+[%collapsible]
+=====
+[source,yaml]
+----
+kind: HorizontalPodAutoscaler
+apiVersion: autoscaling/v2beta2 <1>
+metadata:
+  name: rails-postgresql-example-1
+  namespace: aaa
+  uid: b5be86bd-06bc-4921-a9f2-f421e89b6529
+  resourceVersion: '70256'
+  creationTimestamp: '2021-05-12T17:55:34Z'
+spec:
+  scaleTargetRef:
+    kind: ReplicationController <2>
+    name: rails-postgresql-example-1 <3>
+    apiVersion: v1 <4>
+  minReplicas: 1 <5>
+  maxReplicas: 3 <6>
+  metrics: <7>
+    - type: Resource
+      resource:
+        name: cpu <8>
+        target:
+          type: Utilization <9>
+          averageUtilization: 50 <10>
+----
+<1> Use the `autoscaling/v2beta2` API.
+<2> The kind of object to scale, either `ReplicationController` or `DeploymentConfig`.
+<3> The name of the object to scale. The object must exist.
+<4> The API version of the object to scale:
+* For a replication controller, use v1,
+* For a DeploymentConfig object, use apps.openshift.io/v1.
+<5> The minimum number of replicas when scaling down.
+<6> The maximum number of replicas when scaling up.
+<7> The metrics parameter for memory utilization.
+<8> Use `cpu` for CPU utilization.
+<9> Use `Utilization`.
+<10> Use `averageUtilization` with the targeted CPU value.
+=====
+====
 
 ** To scale for a specific CPU value, create a YAML file similar to the following for an existing `DeploymentConfig` object or replication controller:
 +
```

Comment:
I feel like the following YAML that previously existed can be considered sufficient to enable a customer to use GitOps with a HPA.  I feel like we can remove the two TIPs that were added because the existing documentation is sufficiently clear and the addition of the TIPs does make the customers choices less clear.